frontend/src/app/workflows/workflows.component.html

```diff
@@ -0,0 +1,53 @@
+<div class="h-100 d-flex flex-column">
+  <div class="btn-toolbar align-items-center p-3" role="toolbar" aria-label="Toolbar">
+    <div ngbDropdown class="d-inline-block me-2">
+      <button class="btn btn-primary" id="exampleSelection"
+              ngbDropdownToggle [disabled]="loading">{{currentExampleDesc}}</button>
+      <div ngbDropdownMenu aria-labelledby="exampleSelection">
+        <ng-container *ngFor="let example of examplesList; index as i">
+          <button ngbDropdownItem (click)="changeExampleContent(i)">{{example}}</button>
+        </ng-container>
+      </div>
+    </div>
```

Comment:
`<select>` is probably a more natural choice here instead of a dropdown, which is more fitting for menus with actions.

Reply:
aaaand also done